lib/rules/no-control-regex.js

```diff
@@ -83,8 +96,10 @@ module.exports = {
                 if (regex) {
                     const computedValue = regex.toString();
 
-                    if (hasControlCharacters(computedValue)) {
-                        context.report(node, "Unexpected control character in regular expression.");
+                    const controlCharacters = getControlCharacters(computedValue);
+
+                    if (controlCharacters.length > 0) {
+                        context.report(node, "Unexpected control character(s) in regular expression: " + controlCharacters.join(", ") + ".");
```

Comment:
Let me see if I understand you correctly. You are saying that if we report position, and the same control character appears twice in the same literal, we should report two errors? Hmm... that's a good point. Maybe we can merge this in as it is, and address positioning later?